src/06-loops.js

```diff
@@ -39,8 +55,14 @@ function rangeOdd(min, max) {}
  *  rangeEveryOther(11, 18);
  *  //> [ 11, 13, 15, 17 ]
  */
-function rangeEveryOther(min, max) {}
-
+ function rangeEveryOther(min, max) {
+  let arr=[];
+  for(let i = min;i<=max;i++ ){
```

Comment:
Remember our increment in our for loops doesn't _have_ to be `var++` or `var--`, we can increment however we want. In this case, instead of using `i++` I might have used `i+=2`.